examples/web_sys/webgl/README.md

```diff
@@ -0,0 +1,5 @@
+## Yew WebGL Demo
+
+This is a simple demo using WebGL with Yew to initialize the GL context, create a render loop, and draw to the canvas with basic shaders.
+
+The `stdweb` and `webgl_stdweb` crates are required.
```

Comment:
nit: please update

Reply:
Ahh, sorry I missed converting that. Thanks for the catch. Fixed.